templates/main.tftpl

```diff
@@ -1 +1,34 @@
-#This file is where the code for your application will be in the future.
\ No newline at end of file
+resource "azurerm_resource_group" "network" {
+  name     = "rg-$${local.application_name}Network-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
+
+resource "azurerm_resource_group" "backup" {
+  name     = "rg-$${local.application_name}Backup-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
+
+resource "azurerm_resource_group" "security" {
+  name     = "rg-$${local.application_name}Security-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
+
+resource "azurerm_resource_group" "app" {
+  name     = "rg-$${local.application_name}Application-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
+resource "azurerm_resource_group" "database" {
+  name     = "rg-$${local.application_name}Database-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
+
+resource "azurerm_resource_group" "fileshare" {
+  name     = "rg-$${local.application_name}Files-$${local.stage}-$${local.env_num}"
+  location = local.default_location
+  tags     = local.tags
+}
```

Comment:
Same for this RG

Reply:
We prefer tu use for global standard and we have the code comment for use or delete before deploy.